assets/css/woocommerce/woocommerce.scss

```diff
@@ -1831,8 +1802,9 @@ p.stars {
 			width: 1em;
 			height: 1em;
 			line-height: 1;
-			font-family: star;
-			content: "\53";
+			font-family: "Font Awesome 5 Free" !important;
+			content: "\f005" !important;
```

Comment:
Do these need to be important? I've removed it and the reviews stars are still showing up fine using FontAwesome 5. I wouldn't say this is blocking though, but if you _can_ remove important then I think we should.

Reply:
If you test again but hover over the review stars you should see "S" appear instead of the icon—the woo core styles seem to take precedence for me?

Reply:
Not for me, I just downloaded Edge to see if it was Edge-specific, but they're working perfectly there too.

These are the styles applied to the element:
<img width="328" alt="Screenshot 2021-01-07 at 14 10 22" src="https://user-images.githubusercontent.com/5656702/103902255-2a7bdb80-50f2-11eb-9ee1-01a7f5a79bd8.png">

Reply:
I was seeing the same as @opr so I went ahead and removed the `!important` rules: 5ee80148326d8d6bc15f719119b2ebfbc2e5cf11.